docs/reference/queries.txt

```diff
@@ -245,6 +232,33 @@ Since `id` and `_id` fields are aliases, either one can be used for queries:
   #   embedded: false>
 
 
+Embedded Documents
```

Comment:
I just copy-pasted the "Embedded Documents" section after "Fields" here w/out edits to it, as I felt it made more sense order-wise vis-a-vis additions to Fields section in this PR.